addons/hr_work_entry_contract/models/hr_contract.py

```diff
@@ -192,6 +192,9 @@ def _get_contract_work_entries_values(self, date_start, date_stop):
             if contract.has_static_work_entries() or not leaves:
                 # Empty leaves means empty real_leaves
                 real_leaves = attendances - real_attendances
+            elif not calendar:
```

Comment:
Maybe we could add a test, if it is possible. 😇

Reply:
Yeah I saw, I am just wondering if it was possible to add a test in the same module than the one you did the fix. 🤷‍♂️ 😃